bedrock/firefox/templates/firefox/os/ver/2.5.html

```diff
@@ -0,0 +1,109 @@
+{% endblock %}
+
+{% block features_headline %}
+  <h2>{{_('The Web is the platform')}}</h2>
+{% endblock %}
+
+{% block features_copy %}
+  <p>{{_('Firefox OS is the first truly open mobile platform built entirely on Web technologies. If you can build for the Web using HTML and CSS, you can build Firefox OS.')}}
+  <ul class="features">
+    <li class="customization">
+      <h3>{{_('The most customization')}}</h3>
+      <p>{{_('Make Firefox OS your own with apps and add-ons from the Firefox Marketplace, and keep all your favorite items front and centre with Pin the Web.')}}</p>
+    </li>
+    <li class="control">
+      <h3>{{_('The most control')}}</h3>
+      <p>{{_('From Do Not Track to Private Browsing with Tracking Protection, Firefox OS is designed keep you in control of your browsing activity.')}}</p>
+    </li>
+  </ul>
+{% endblock %}
+
+{% block fxox_apps %}
+{% endblock %}
+
+{% block fxox_news %}
+{% endblock %}
+
+{% block fxox_marketplace %}
+{% endblock %}
+
+{% block fxox_hub %}
+  <section class="fxos-hub">
+    <div class="content-wrapper">
+      <div class="hub-container">
+        <h2>{{_('Visit the Participation Hub')}}</h2> 
+        <p>{{_('Want to make Firefox OS even better? Get all the resources, tools and documentation you need from the Participation Hub.')}}</p>
+        <p><a class="more" href="#TODO">{{_('Check it out')}}</a></p>
+      </div>
+      <div class="hub-image">
+        {{ high_res_img('firefox/os-new/hub.png', {'width': '241', 'height': '261'}) }}
+      </div>
+    </div>
+  </section>
+{% endblock %}
+
+{% block fxox_mdn %} 
+  <section class="fxos-mdn">
+    <div class="content-wrapper">
+      <div class="mdn-container">
+        <h2>{{_('Learn more at MDN')}}</h2>
+        <p>{{_('Want to do even more with Firefox OS? Head over to the Mozilla Developer Network to learn more about platform guides, Firefox OS add-ons and more.')}}</p>
+        <p><a class="more" href="#TODO">{{_('Visit MDN')}}</a></p>
+      </div>
+      <div class="mdn-image">
+        {{ high_res_img('firefox/os-new/mdn-logo.png', {'width': '301', 'height': '304'}) }}
+      </div>
+    </div>
+  </section>
+{% endblock %}
+
+{% block fxox_community %}
+{% endblock %}
+
+{% block fxox_help %}
+{% endblock %}
+
+{% block fxox_try %}
+  <section class="fxos-try">
+    <div class="content-wrapper">
+      <h2>{{_('Ready to try Firefox OS 2.5 Developer Preview on your Android device?')}}</h2>
+      <div class="primary-cta-signup signup">
+        <a href="#TODO" class="cta-button" role="button">{{_('Get the Android App')}}</a>
+        <p>{{_('Want the full Firefox OS 2.5 Developer Preview experience?')}} <a href="#TODO">{{_('Flash your phone »')}}</a></p>
+      </div>
+    </div>
+  </section>
+{% endblock %}
+
+{% block email_form %}
+  <section class="fxos-signup">
+    <div class="content-wrapper">
+      {{ email_newsletter_form(title="Sign up to get the latest Firefox OS news and more", submit_text="Sign up now") }}
```

Comment:
No, I left it out because I was getting errors adding the l10n block inside. @alexgibson Wondering what the syntax for that would be?